src/main/java/bio/terra/workspace/service/workspace/WorkspaceCloudContext.java

```diff
@@ -1,24 +1,42 @@
 package bio.terra.workspace.service.workspace;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.google.auto.value.AutoValue;
 import java.util.Optional;
 
 /**
  * The cloud contexts associated with a resource.
  *
+ * <p>This uses AutoValue's builder pattern to ensure it's serializable with Jackson. Otherwise,
+ * Jackson doesn't understand how to deserialize AutoValue classes.
+ *
  * <p>A workspace can have resources with each cloud provider, its "cloud context."
  * <li>For GCP, a Google Project is associated with every resource.
  */
 @AutoValue
+@JsonSerialize(as = WorkspaceCloudContext.class)
```

Comment:
I am not a fan of AutoValue. IMO the obscurity outweighs the benefit.

Reply:
I think it definitely adds complexity (especially combined with Jackson), but I also think the benefit of generating correct value classes is nontrivial. It's also used for most value classes in WM - it sounds like it's worth a higher level discussion, but I think changing it is out of scope for this PR.

Reply:
ack. Tracking this in PF-433